appservice/src/deploy/getDeployFsPath.ts

```diff
@@ -127,8 +127,7 @@ function getContainingWorkspace(context: IActionContext, fsPath: string): vscode
             });
         });
 
-        context.telemetry.properties.cancelStep = 'openInNewWindow';
```

Comment:
'cancelStep' is reset to undefined in some places but just removed in others?

Reply:
Well, you're describing the old behavior and yes it was a bit weird/inconsistent. The new behavior does not use `cancelStep` at all

Reply:
oh right, didn't notice.